packages/audio-block/src/AudioBlock.tsx

```diff
@@ -186,32 +198,58 @@ export const AudioBlock = ({ appBridge }: BlockProps) => {
             )}
             <div className="tw-flex tw-gap-4 tw-justify-between tw-w-full">
                 <div className="tw-flex-1">
-                    <div data-test-id="audio-block-title">
-                        <RichTextEditor
-                            designTokens={designTokens}
-                            border={false}
-                            onBlur={saveTitle}
-                            placeholder={isEditing ? 'Asset name' : undefined}
-                            readonly={!isEditing}
-                            value={title ?? DEFAULT_CONTENT_TITLE}
-                            plugins={customTitlePlugins}
-                        />
-                    </div>
-                    <div data-test-id="audio-block-description">
-                        <RichTextEditor
-                            designTokens={designTokens}
-                            border={false}
-                            position={Position.FLOATING}
-                            onBlur={saveDescription}
-                            placeholder={isEditing ? 'Add a description here' : undefined}
-                            readonly={!isEditing}
-                            value={description ?? DEFAULT_CONTENT_DESCRIPTION}
-                        />
-                    </div>
+                    {title !== DEFAULT_CONTENT_TITLE || isEditing ? (
+                        <div>
+                            {!isEditing ? (
+                                <>
+                                    {hasRichTextValue(title) && (
+                                        <div
+                                            data-test-id="block-title-html"
+                                            dangerouslySetInnerHTML={{ __html: htmlTitle }}
+                                        />
+                                    )}
+                                </>
+                            ) : (
+                                <RichTextEditor
+                                    designTokens={designTokens}
+                                    border={false}
+                                    onBlur={saveTitle}
+                                    placeholder="Asset name"
+                                    value={title ?? DEFAULT_CONTENT_TITLE}
+                                    plugins={customTitlePlugins}
+                                    updateValueOnChange
+                                />
+                            )}
+                        </div>
+                    ) : null}
+
+                    {description !== DEFAULT_CONTENT_DESCRIPTION || isEditing ? (
+                        <div>
+                            {!isEditing ? (
```

Comment:
do you need both conditions & the `hasRichTextValue(description)`?

Reply:
no for sure not, removed the bad ones

Reply:
Looks good like this, also in the UI.